.azure/infrastructure/main.bicep

```diff
@@ -48,6 +48,16 @@ param appInsightsPurgeDataOn30Days bool = false
 @description('Whether zone redundancy should be enabled for the container app environment')
 param containerAppEnvZoneRedundancyEnabled bool
 
+@description('Workload profiles to enable in the container app environment')
+param workloadProfiles array = [
+  {
+    name: 'Consumption'
+    workloadProfileType: 'Consumption'
+    minimumCount: 0
+    maximumCount: 10
```

Comment:
```
    minimumCount: 0
    maximumCount: 10
```

This might not be needed for Consumption type, because I don't see how that would make any sense 🤔

Reply:
I think minimumCount and maximumCount can (and maybe even should) be omitted when specifying the Consumption workload profile (see eg [this](https://github.com/microsoft/azure-container-apps/issues/731))